### ISSUE TITLE
Failed to run local testnet: No valid eth1_data votes

### ISSUE THREAD

**Issue by bastienfaivre** [sigp/lighthouse#4536]
## Description

I'm trying to run the local testnet provided in the repository. Without doing any modifications to the configurations, I executed `./start_local_testnet.sh genesis.json`. Everything seems to work fine except the beacon nodes. Indeed, I still get this error:
```
Jul 24 16:01:15.009 ERRO No valid eth1_data votes, `votes_to_consider` empty, outcome: casting `state.eth1_data` as eth1 vote, genesis_time: 1690214400, earliest_block_timestamp: None, lowest_block_number: None, service: deposit_contract_rpc
```
I read issue #1066 and tried to change the genesis.json timestamp to something more recent, or `0`. I also tried to change the `GENESIS_DELAY` variable. But I still get this error.

## Version

I'm using the currently [last stable commit](https://github.com/sigp/lighthouse/tree/dfcb3363c757671eb19d5f8e519b4b94ac74677a)

## Present Behaviour

Here is the error in context:
```
Jul 24 16:01:10.503 WARN Syncing deposit contract block cache    est_blocks_remaining: initializing deposits, service: slot_notifier
Jul 24 16:01:10.503 INFO Ready for Capella                       info: ensure the execution endpoint is updated to the latest Capella/Shanghai release, service: slot_notifier
Jul 24 16:01:12.003 INFO Prepared beacon proposer                parent_root: 0x05d6e384a17790d9acb921d373e0839f34af0ffd1e6d84676b5486a3a2934445, validator: 75, prepare_slot: 25, service: beacon
Jul 24 16:01:12.058 INFO New block received                      root: 0x380c5ff3e7d7756423c4fd1b5fde951874f1d613224ac27811a992c40ebb3662, slot: 24
Jul 24 16:01:12.100 INFO Prepared beacon proposer                parent_root: 0x380c5ff3e7d7756423c4fd1b5fde951874f1d613224ac27811a992c40ebb3662, validator: 75, prepare_slot: 25, service: beacon
Jul 24 16:01:13.501 INFO Synced                                  slot: 24, block: 0x380c…3662, epoch: 0, finalized_epoch: 0, finalized_root: 0x850f…0a2e, exec_hash: 0x216b…f436 (verified), peers: 3, service: slot_notifier
Jul 24 16:01:13.501 WARN Syncing deposit contract block cache    est_blocks_remaining: initializing deposits, service: slot_notifier
Jul 24 16:01:13.501 INFO Ready for Capella                       info: ensure the execution endpoint is updated to the latest Capella/Shanghai release, service: slot_notifier
Jul 24 16:01:15.009 ERRO No valid eth1_data votes, `votes_to_consider` empty, outcome: casting `state.eth1_data` as eth1 vote, genesis_time: 1690214400, earliest_block_timestamp: None, lowest_block_number: None, service: deposit_contract_rpc
Jul 24 16:01:15.046 INFO Signed block published to network via HTTP API, publish_delay: 1.947929ms, slot: 25
Jul 24 16:01:15.072 INFO Valid block from HTTP API               slot: 25, proposer_index: 75, root: 0xa494…f166, block_delay: 44.935485ms
Jul 24 16:01:16.502 INFO Synced                                  slot: 25, block: 0xa494…f166, epoch: 0, finalized_epoch: 0, finalized_root: 0x850f…0a2e, exec_hash: 0xa47a…6baa (verified), peers: 3, service: slot_notifier
```

## Expected Behaviour

No error

## Steps to resolve

Given a freshly cloned repository, I would like to know what changes should be applied to the actual code to get a fully functional testnet. Also, how do these changes make the testnet work compared to the current state. Thank you in advance for your help

**Comment by chong-he** [comment]
We found that a `ETH1_FOLLOW_DISTANCE` of 1 as defined in `setup.sh` is too low, and revised it to 128. We also revise SECONDS_PER_ETH1_BLOCK and set it to be the same as the consensus layer slot time (3 seconds)

After testing, the error
```
ERRO No valid eth1_data votes, `votes_to_consider` empty, outcome: casting `state.eth1_data` as eth1 vote, genesis_time: 1690214400, earliest_block_timestamp: None, lowest_block_number: None, service: deposit_contract_rpc
```

will go away after ~2 hours and the local blockchain is progressing normally. 

The fix is in #4749, feel free to try it out and let us know how it goes

**Comment by jimmygchen** [comment]
Fixed in #4749  🎉
